openxr/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub use passthrough::*;
 pub use builder::{
     CompositionLayerBase, CompositionLayerCubeKHR, CompositionLayerCylinderKHR,
     CompositionLayerEquirectKHR, CompositionLayerProjection, CompositionLayerProjectionView,
-    CompositionLayerQuad, HapticBase, HapticVibration, SwapchainSubImage,
+    CompositionLayerQuad, HapticBase, HapticPcmVibrationFB, HapticVibration, SwapchainSubImage,
```

Comment:
These are easy to forget. Can we use `builder::*` instead?

Reply:
Here is an incomplete list of things we are not exporting from `builder`:

Reply:
IIRC this is currently intended to be an allowlist: we only export builders that the high-level API consumes.